setup.py

```diff
@@ -55,18 +55,15 @@
     "tiktoken",
     "sentence-transformers",
     "open_clip_torch>=2.26.1",
+    "peft",
 ]
 
 QUALITY_REQUIRE = ["black~=23.1", "ruff==0.4.4"]
 
 EXTRAS_REQUIRE = {
-    "neural-compressor": ["neural-compressor[pt]>3.0", "accelerate", "transformers<4.46"],
-    "openvino": [
-        "openvino==2024.4.1.dev20240926",
-        "nncf>=2.11.0",
-        "openvino-tokenizers[transformers]==2024.4.1.0.dev20240926",
-    ],
     "nncf": ["nncf>=2.11.0"],
+    "openvino": ["nncf>=2.11.0", "openvino==2024.4.1.dev20240926", "openvino-tokenizers==2024.4.1.0.dev20240926"],
```

Comment:
is there a reason why these versions are pinned ?

Reply:
yes, it is done because issues with openvino 2024.4.0, 2024.4.1 contains fixing for it, but this package is not released yet, prerelease packages can not be installed in pip without --pre (this approach does not work with setup.py) or with strict version. We have IR to unpin it when release package will be released